docs/src/operate/registration/mqtt_api.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Entity management REST APIs
+tags: [Child-Device, Registration]
+sidebar_position: 1
+description: Register child devices and services with %%te%% MQTT APIs
+---
+
+# MQTT APIs for Entity Management
+
+The entity management MQTT APIs of %%te%% allows MQTT clients to register and deregister entities (child devices and services).
```

Comment:
I would try to keep at least the first paragraphs of this section in sync with the HTTP APIs, the point being to help a user choose the more appropriate approach for their use case.

Notably, the HTTP sections highlights the advantage of using HTTP. I would do the same here and tell when MQTT is appropriate.

For example:

```
%%te%% provides an MQTT API to manage all the entities (devices and services) attached to a main device. This interface let you create, update and delete entities as well as observe changes.

When compared to the HTTP APIs, the MQTT API shines when the point is to react to entity related changes as new child devices, new services or metadata updates. 
```

Reply:
Resolved by 4ff49e7